pkg/resourcemonitor/podresourcesscanner.go

```diff
@@ -198,7 +202,22 @@ func (resMon *PodResourcesScanner) Scan() ([]PodResources, error) {
 
 	}
 
-	return podResData, nil
+	if resMon.podFingerprint && len(podResData) > 0 {
+		var status podfingerprint.Status
+		podFingerprintSign, err := computePodFingerprint(podResData, &status)
+		if err != nil {
+			klog.Errorf("podFingerprint: Unable to compute fingerprint %v", err)
```

Comment:
should we fail badly or should we update without the annotation? If it's an annotation, it seems excessive to fail because of this. I'll think more about this.

Reply:
Yeah ... it somehow seems too much to fail due to an annotation but in the end I went for the failure just because the user has to explicitly activate this feature and I slightly prefer to fail ... but I had the same doubt